src/model.cc

```diff
@@ -96,7 +96,7 @@ Model::Init()
     const auto config_path = JoinPath({model_dir_, kModelConfigPbTxt});
     bool exists = false;
     RETURN_IF_ERROR(FileExists(config_path, &exists));
-    if (!exists) {
+    if (!exists && !is_config_override_) {
```

Comment:
Instead of passing `is_config_override_`, you can pass a generic flag `is_config_provided_` which is `config_override | config_file_exisits`. In fact, you can pass `parsed_config` in `ModelRepositoryManager::InitializeModelInfo()` which should have the same evaluation as what is mentioned above.

Reply:
Updated, please review.